### PR TITLE
faq: add PD encryption error "fail to get encryption key from file"

### PR DESCRIPTION
**Pull request by RobertCheng-956**
### What is changed, added or deleted? (Required)

faq: add PD encryption error "fail to get encryption key from file"

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [ ] v7.6 (TiDB 7.6 versions)
- [x] v7.5 (TiDB 7.5 versions)
- [x] v7.4 (TiDB 7.4 versions)
- [x] v7.3 (TiDB 7.3 versions)
- [x] v7.1 (TiDB 7.1 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [x] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/12506
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by RobertCheng-956** [comment]
translation from https://github.com/pingcap/docs-cn/pull/15416

**Review by dveeden — APPROVED** [review]
The `%!(EXTRA` indicates a formatting error in the code. That probably needs to be fixed as well. However that is outside of the scope of this PR

**Comment by RobertCheng-956** [comment]
I don't know why Jenkins job failed.

**Comment by dveeden** [comment]
> I don't know why Jenkins job failed.

I think it might be due to this GitHub incident: https://www.githubstatus.com/incidents/pxg3dz4yg7lp

**Comment by RobertCheng-956** [comment]
> > I don't know why Jenkins job failed.
> 
> 
> 
> I think it might be due to this GitHub incident: https://www.githubstatus.com/incidents/pxg3dz4yg7lp

So what should I do to fix this problem? I haven't met this kind of problem yet.

**Comment by dveeden** [comment]
> > > I don't know why Jenkins job failed.
> > 
> > 
> > I think it might be due to this GitHub incident: https://www.githubstatus.com/incidents/pxg3dz4yg7lp
> 
> So what should I do to fix this problem? I haven't met this kind of problem yet.

My `/verify` comment earlier triggers the test to run again.

**Comment by RobertCheng-956** [comment]
> > > > I don't know why Jenkins job failed.
> 
> > > 
> 
> > > 
> 
> > > I think it might be due to this GitHub incident: https://www.githubstatus.com/incidents/pxg3dz4yg7lp
> 
> > 
> 
> > So what should I do to fix this problem? I haven't met this kind of problem yet.
> 
> 
> 
> My `/verify` comment earlier triggers the test to run again.

Ok, I got. Thank you for your detailed replies. This is almost my first time to contribute, so I am a little bit confused when some errors occur.

**Comment by rpaik** [comment]
> > > > > I don't know why Jenkins job failed.
> > 
> > 
> > > >
> > 
> > 
> > > >
> > 
> > 
> > > > I think it might be due to this GitHub incident: https://www.githubstatus.com/incidents/pxg3dz4yg7lp
> > 
> > 
> > > 
> > 
> > 
> > > So what should I do to fix this problem? I haven't met this kind of problem yet.
> > 
> > 
> > My `/verify` comment earlier triggers the test to run again.
> 
> Ok, I got. Thank you for your detailed replies. This is almost my first time to contribute, so I am a little bit confused when some errors occur.

Thanks for contributing to TiDB Docs Dash and welcome to the community @RobertCheng-956!

**Comment by RobertCheng-956** [comment]
OK. Thank you

File: faq/manage-cluster-faq.md [hunk 1]
```diff
@@ -145,6 +145,10 @@ Most of the APIs of PD are available only when the TiKV cluster is initialized.
 
 This is because the `--initial-cluster` in the PD startup parameter contains a member that doesn't belong to this cluster. To solve this problem, check the corresponding cluster of each member, remove the wrong member, and then restart PD.
 
+### Error when Enabling Static Encryption in PD: `[PD:encryption:ErrEncryptionNewMasterKey]fail to get encryption key from file /root/path/file%!(EXTRA string=open /root/path/file: permission denied)`
```
Comment (Oreoxmt) [inline faq/manage-cluster-faq.md:148]:
```suggestion
### The `[PD:encryption:ErrEncryptionNewMasterKey]fail to get encryption key from file /root/path/file%!(EXTRA string=open /root/path/file: permission denied)` message is displayed when enabling encryption at rest for PD
```

File: faq/manage-cluster-faq.md [hunk 2]
```diff
@@ -145,6 +145,10 @@ Most of the APIs of PD are available only when the TiKV cluster is initialized.
...
+ 
+Static encryption does not support storing the key file in the `root` directory or its subdirectories. Even with added read permissions, the same error will occur. If you encounter this issue, consider placing the key file in a path outside of the `root` directory.
```
Comment (Oreoxmt) [inline faq/manage-cluster-faq.md:150]:
```suggestion
Encryption at rest does not support storing the key file in the `root` directory or its subdirectories. Even if you grant read permissions, the same error occurs. To resolve this issue, store the key file in a location outside the `root` directory.
```
https://docs.pingcap.com/tidb/stable/encryption-at-rest

**Review by Oreoxmt — APPROVED** [review]

**Pull request merged**